examples/pes/pswater/nvt/input.xml

```diff
@@ -5,7 +5,9 @@
     <trajectory filename='vel' format='xyz' stride='5'> velocities </trajectory>
     <trajectory filename='for' format='xyz' stride='2'> forces </trajectory>
     <trajectory filename='kin' format='xyz' stride='2'> kinetic_cv </trajectory>
-    <trajectory filename='dip' stride='2'> extras </trajectory>
+    <trajectory filename='dip' stride='2' extra_type='dipole'> extras </trajectory>
+    <trajectory filename='raw' stride='2' extra_type='raw'> extras </trajectory>    
+    <trajectory filename='dip0' stride='2' extra_type='dipole'> extras_component(0) </trajectory>
```

Comment:
Also here, can we be a bit more verbose about what this does, at least in the examples input file? And also here the example is even harder to find :-D